README.md

```diff
@@ -73,7 +70,7 @@ TODOs
 
 Contributing as a Developer
 --------
-Please see the Contributing.md file before being any new work. 
+Please see the Contributing.md file before being any new work.
```

Comment:
I think this is a setting of sublime text that is been applied to this project for whatever reason. I ll look at it later.